src/server.ts

```diff
@@ -3,18 +3,11 @@ import { search } from './search.ts'
 import { genQuery } from './genQuery.ts'
 import { getBrowser, getNewContext } from './context.ts'
 import * as gptscript from '@gptscript-ai/gptscript'
+import { rmSync } from 'node:fs'
 
 const gptsClient = new gptscript.GPTScript()
 
-const input = process.env.GPTSCRIPT_INPUT
-delete (process.env.GPTSCRIPT_INPUT)
-if (input === undefined) {
-  console.log('error: no input provided')
-  process.exit(1)
-}
-
-const data = JSON.parse(input)
-const question: string = data.question ?? ''
+const question: string = process.env.QUESTION ?? ''
```

Comment:
It didn't make sense to unmarshal the JSON from `GPTSCRIPT_INPUT` like we were doing before, when we can just read it straight from this environment variable.